.rubocop.yml

```diff
@@ -100,6 +101,168 @@ Packaging/RequireHardcodingLib:
 Packaging/RequireRelativeHardcodingLib:
   Enabled: true
 
+Performance:
+  Enabled: true
+
+Performance/AncestorsInclude:
+  Enabled: false
+
+Performance/ArraySemiInfiniteRangeSlice:
+  Enabled: false
+
+Performance/BigDecimalWithNumericArgument:
+  Enabled: true
+
+Performance/BindCall:
+  Enabled: true
+
+Performance/BlockGivenWithExplicitBlock:
+  Enabled: true
+
+Performance/Caller:
+  Enabled: true
+
+Performance/CaseWhenSplat:
+  Enabled: true
+
+Performance/Casecmp:
+  Enabled: false
+
+Performance/ChainArrayAllocation:
+  Enabled: false
+
+Performance/CollectionLiteralInLoop:
+  Enabled: true
+  Exclude:
+    - features/**/*
+    - spec/**/*
```

Comment:
No need to exclude these since there are no violations currently so we can enforce for now.

```suggestion
```

Reply:
There are four violations:

```
spec/unit/resource_controller/polymorphic_routes_spec.rb:42:9: C: Performance/CollectionLiteralInLoop: Avoid immutable Array literals in loops. It is better to extract it into a local variable or a constant.
        %w(posts user_posts).each do |current_page|
        ^^^^^^^^^^^^^^^^^^^^
spec/unit/resource_controller/polymorphic_routes_spec.rb:87:9: C: Performance/CollectionLiteralInLoop: Avoid immutable Array literals in loops. It is better to extract it into a local variable or a constant.
        %w(posts category_posts user_posts).each do |current_page|
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/unit/views/components/attributes_table_spec.rb:78:11: C: Performance/CollectionLiteralInLoop: Avoid immutable Array literals in loops. It is better to extract it into a local variable or a constant.
          [ ...
          ^
spec/unit/views/components/attributes_table_spec.rb:196:48: C: Performance/CollectionLiteralInLoop: Avoid immutable Array literals in loops. It is better to extract it into a local variable or a constant.
            expect(col.class_names).to include(["even", "odd"][index % 2])
```